### PR TITLE
docs: keyman 14 upgrade and keyboard source download KBs

### PR DESCRIPTION
**Pull request by mcdurdin**
Two knowledge base articles:

* KMKB0105: INFO: Potential issues when upgrading to Keyman for Windows 14 with custom language profiles
* KMKB0106: HOWTO: Download a single keyboard folder from the Keyboards repository

### CODE REVIEW

**Comment by DavidLRowe** [comment]
In KMKB0106, the SVN command should be:
```
svn export https://github.com/keymanapp/keyboards/trunk/release/sil/sil_ipa
```
because SVN has a special name for the master branch.

**Comment by mcdurdin** [comment]
The original command is correct, not the one with `trunk` :)

```
C:\temp\test>svn export https://github.com/keymanapp/keyboards/trunk/release/sil/sil_ipa
svn: E170000: URL 'https://github.com/keymanapp/keyboards/trunk/release/sil/sil_ipa' doesn't exist
```

```
C:\temp\test>svn export https://github.com/keymanapp/keyboards/branches/master/release/sil/sil_ipa
A    sil_ipa
A    sil_ipa\HISTORY.md
A    sil_ipa\LICENSE.md
<snip>
```

File: knowledge-base/kb0105.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+| `grc-Grek-GR` | Galaxie Greek Mnemonic
+
+This has arisen because the upgrade tried to move the Tok Pisin keyboard from
+`tpi-Latn` to `tpi-PG`, as the `tpi` language code should not specify the `Latn`
+writing system according to canonicalization rules.
+
+### Workaround
+
+You can have up to 4 transient language codes in Windows. Choose the codes you
+wish to use, remove the others, and you should be able to install the language
```
Comment (darcywong00) [inline knowledge-base/kb0105.md:59]: > remove the others

Is that done in Keyman or Windows settings?

Comment (mcdurdin) [reply knowledge-base/kb0105.md:59]: Either ...

**Comment by mcdurdin** [comment]
Turns out v1.12.0 of svn uses the `branches/master` url but v1.9 uses `trunk`. Yuck! Recommendation: use v1.12.0...

**Pull request merged**